tests/run-make/used/rmake.rs

```diff
@@ -0,0 +1,14 @@
+use run_make_support::{rustc, tmp_dir};
+use std::process::Command;
+
+fn main() {
+    rustc().opt_level("3").emit("obj").input("used.rs").run();
+
+    let output = Command::new("nm").arg(tmp_dir().join("used.o")).output().unwrap();
```

Comment:
Suggestion: we should probably not rely on having `nm` in the environment. https://github.com/rust-lang/rust/pull/125787 implements `nm` on top of third-party libraries, maybe we use that?

Reply:
If going with external `nm`, then this could be updated to use the new APIs:

```rs
cmd("nm").arg("used.o").run().assert_stdout_contains("FOO");
```